src/Makefile.am.inc

```diff
@@ -39,6 +40,7 @@ $(xdp_dbus_built_sources) : $(FLATPAK_IFACE_FILES) $(PORTAL_IFACE_FILES)
 	$(AM_V_GEN) $(GDBUS_CODEGEN)                            \
 		--interface-prefix org.freedesktop.portal.      \
 		--c-namespace Xdp                               \
+                --c-generate-autocleanup all                      \
```

Comment:
This adds a dependency on a *very* recent glib. I'm not sure we want to do that. In flatpak I instead manually define autocleanups for the dbus types.